sdk/monitor/Azure.Monitor.Query/api/Azure.Monitor.Query.netstandard2.0.cs

```diff
@@ -1,9 +1,31 @@
+namespace Azure.Core
+{
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct MonitorQueryDateTimeRange : System.IEquatable<Azure.Core.MonitorQueryDateTimeRange>
+    {
+        public MonitorQueryDateTimeRange(System.DateTimeOffset start, System.DateTimeOffset end) { throw null; }
+        public MonitorQueryDateTimeRange(System.DateTimeOffset start, System.TimeSpan duration) { throw null; }
+        public MonitorQueryDateTimeRange(System.TimeSpan duration) { throw null; }
+        public MonitorQueryDateTimeRange(System.TimeSpan duration, System.DateTimeOffset end) { throw null; }
+        public static Azure.Core.MonitorQueryDateTimeRange All { get { throw null; } }
+        public System.TimeSpan Duration { get { throw null; } }
+        public System.DateTimeOffset? End { get { throw null; } }
+        public System.DateTimeOffset? Start { get { throw null; } }
+        public bool Equals(Azure.Core.MonitorQueryDateTimeRange other) { throw null; }
+        public override bool Equals(object obj) { throw null; }
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.Core.MonitorQueryDateTimeRange left, Azure.Core.MonitorQueryDateTimeRange right) { throw null; }
+        public static implicit operator Azure.Core.MonitorQueryDateTimeRange (System.TimeSpan timeSpan) { throw null; }
+        public static bool operator !=(Azure.Core.MonitorQueryDateTimeRange left, Azure.Core.MonitorQueryDateTimeRange right) { throw null; }
+        public override string ToString() { throw null; }
```

Comment:
I noticed the `ToIsoString` method is internal.  We should revisit that if we ever end up adding a `Parse` method for ISO ranges.